cirq-ionq/cirq_ionq/ionq_gateset.py

```diff
@@ -78,12 +78,20 @@ def _decompose_two_qubit_operation(self, op: cirq.Operation, _) -> cirq.OP_TREE:
             temp, k=1, rewriter=lambda op: self._decompose_single_qubit_operation(op, -1)
         ).all_operations()
 
+    def _decompose_multi_qubit_operation(self, op: cirq.Operation, _) -> cirq.OP_TREE:
+        if isinstance(op.gate, cirq.CCZPowGate):
+            return cirq.ops.decompose_all_to_all_connect_ccz_gate(op.gate, op.qubits)
+        return NotImplemented
+
     @property
     def preprocess_transformers(self) -> List['cirq.TRANSFORMER']:
-        """List of transformers which should be run before decomposing individual operations."""
+        """List of transformers which should be run before decomposing individual operations.
```

Comment:
Add a new line after summary line and before description. 
```suggestion
        """List of transformers which should be run before decomposing individual operations.
       
```